pkg/assets/handlers.go

```diff
@@ -141,6 +144,18 @@ func HTML5ModeHandler(contextRoot string, subcontextMap map[string]string, h htt
 	}), nil
 }
 
+var versionTemplate = template.Must(template.New("webConsoleVersion").Parse(`
+window.OPENSHIFT_VERSION = {
+  openshift: "{{ .OpenShiftVersionInfo.GitVersion | js}}",
+  kubernetes: "{{ .KubernetesVersionInfo.GitVersion | js}}"
+};
+`))
```

Comment:
I would replicate this structure in the static `config.js` and give it a string like "dev-source" or something... just want to keep the `serve-local-assets.sh` env as close as possible to the prod one